### ISSUE TITLE
Slider and target values on card don't update when manually changed.

### ISSUE THREAD

**Issue by CyaOnDaNet** [enriqg9/dual-thermostat#6]
When I walk over to the thermostat and manually change the temperature target values, the card doesn't update the slider or target values. The overall temperature reading in the middle is still accurate. Changing the slider using the card still updates the thermostat. Basically, it's only showing changes made using the card in Home Assistant.

**Comment by enriqg9** [comment]
What version of home assistant? Which thermostat? Cannot reproduce on my end.

**Comment by CyaOnDaNet** [comment]
I'm on Home Assistant version 0.93.1 and its a 2GIG Technologies CT101 Thermostat (Iris).

**Comment by enriqg9** [comment]
This is probably something related to your thermostat, this is happening to 2 others with the same thermostat:

[Link 1](https://community.home-assistant.io/t/dual-thermostat-card/85798/122)
[Link 2](https://community.home-assistant.io/t/dual-thermostat-card/85798/66)

**Comment by enriqg9** [comment]
Have you updated to Hass 0.96? Can you check if this still happens to you on [0.3.0](https://github.com/enriqg9/dual-thermostat/releases/tag/0.3.0)? I'm still not able to replicate.

**Comment by CyaOnDaNet** [comment]
Updating everything now, I’m out of town for the weekend so I won’t be able to test with the physical unit until Sunday night. Thanks.

**Comment by CyaOnDaNet** [comment]
Just an update, this is definitively a problem with my thermostat and not the custom card. I read online somewhere that someone with the same thermostat had to remove and then pair their thermostat with the z-wave network again to get it to function 100%. I have tried this several times and have had many different issues. Interestingly, I have found that there are 2 different naming schemes home assistant will assign when it pairs. I will continue to try to get it added properly and will update here if I was successful just so others with the same thermostat know. Thanks for your help and creation of this very useful card!
